File: pgm/ABC_Visual_Basic/ABC_149_B.vb
' Imports System.Collections.Generic
' Imports System.Text.RegularExpressions

Class Main
    Public Shared Sub Main()
        Dim A, B, K As Long
        Dim arr() As String
        arr = Split(Console.ReadLine())
        A = arr(0) : B = arr(1) : K = arr(2)
        Console.WriteLine(CStr(A - Math.Min(A, K)) & " " & CStr(B - Math.Min(B, K - Math.min(A, K))))
        Return
    End Sub
End Class

File: pgm/ABC_C#/ABC_149_B.cs
using System;
// using System.Collections.Generic;
// using System.Linq;

class Program
{
    static void Main()
    {
        String[] arr = Console.ReadLine().Split();
        long A, B, K;
        A = long.Parse(arr[0]);
        B = long.Parse(arr[1]);
        K = long.Parse(arr[2]);
        Console.WriteLine((A - Math.Min(A, K)).ToString()
            + " "
            + (B - Math.Min(B, K - Math.Min(A, K))).ToString());
        return;
    }
}